example/pubspec.yaml

```diff
@@ -8,7 +8,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 version: 0.0.1
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.12.0-0.0.dev <3.0.0"
```

Comment:
Maybe using a more tolerant constraint such as ">=2.10.0-0 <3.0.0"?

Reply:
It doesn't let me because 2.10 doesn't have support for nndb